### PR TITLE
UPSTREAM: 44970: CRI: Fix StopContainer timeout

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

**Comment by sttts** [comment]
[test]

**Comment by deads2k** [comment]
[merge]

**Comment by deads2k** [comment]
https://github.com/openshift/origin/issues/13942

File: vendor/k8s.io/kubernetes/pkg/kubelet/remote/remote_runtime.go [hunk 1]
```diff
@@ -209,7 +209,11 @@ func (r *RemoteRuntimeService) StartContainer(containerID string) error {
 
 // StopContainer stops a running container with a grace period (i.e., timeout).
 func (r *RemoteRuntimeService) StopContainer(containerID string, timeout int64) error {
-	ctx, cancel := getContextWithTimeout(r.timeout)
+	ctx, cancel := getContextWithTimeout(time.Duration(timeout) * time.Second)
+	if timeout == 0 {
```
Comment (smarterclayton) [inline vendor/k8s.io/kubernetes/pkg/kubelet/remote/remote_runtime.go]: Wait - why do we get a context twice?

**Comment by deads2k** [comment]
re[merge]

**Comment by deads2k** [comment]
https://github.com/openshift/origin/issues/13973

**Pull request merged**